vncdotool/api.py

```diff
@@ -80,6 +80,17 @@ class ThreadedVNCClientProxy(object):
     def __init__(self, factory):
         self.factory = factory
         self.queue = queue.Queue()
+        self._timeout = 60 * 60
+
+    @property
+    def timeout(self):
```

Comment:
What is the motivation behind using a property here rather than a simple data member on the class?

Reply:
1. A bit more future proof. If the timeout needs to be propagated to the client class, for example, this will be handled transparently.
2.  Properties allow to document and introspect the arguments.
    ```
    In [3]: client.timeout?
    Type:        property
    String form: <property object at 0x7f7ed48db548>
    Docstring:   Timeout in seconds for API requests.
    ```

Reply:
thanks for the explanation, I tend to use members and convert them to properties when needed but hadn't considered the documentation aspect before.